README.md

```diff
@@ -3,6 +3,9 @@
 
 # Apostrophe Pieces Exporter
 
+> ⚠️ **NOTE:** This module is a deprecated version of the Import-Export module.  
```

Comment:
Just "this module is deprecated, its functionality has been incorporated into..."

We don't want to suggest it's merely a deprecated version because the entire module is deprecated